spec/requests/schools_request_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe "Schools", type: :request do
+
+  describe "Creating a school" do
+    it "successfully creates a school" do
+      # Given
+      expected_name = "Test School"
```

Comment:
Using let(:expected_name) { "Test School" } etc above the it would be better.

Reply:
Ah good to know. Why is this better?

Reply:
The let blocks are lazily instantiated so objects tend to just get created on demand at run time. And these testing variables tend to be applicable to multiple tests as well. With let blocks you don't duplicate them for every spec. The overall result is that the specs look less cluttered and are much easier to read and understand. https://www.betterspecs.org/#let has some good guidelines for writing ruby specs.